pkg/core/blockchain.go

```diff
@@ -2769,9 +2769,9 @@ var (
 	ErrWitnessHashMismatch         = errors.New("witness hash mismatch")
 	ErrNativeContractWitness       = errors.New("native contract witness must have empty verification script")
 	ErrVerificationFailed          = errors.New("signature check failed")
-	ErrInvalidInvocation           = errors.New("invalid invocation script")
+	ErrInvalidInvocationScript     = errors.New("invalid invocation script")
 	ErrInvalidSignature            = fmt.Errorf("%w: invalid signature", ErrVerificationFailed)
-	ErrInvalidVerification         = errors.New("invalid verification script")
+	ErrInvalidVerificationScript   = errors.New("invalid verification script")
```

Comment:
Adjust the commit message, please, because you're renaming not only the `ErrInvalidVerification` but also `ErrInvalidInvocation`. You can specify something like `core: rename ErrInvalidVerification and ErrInvalidInvocation` in the message header and specify the reason of renaming in the message body: `Use more specific and meaningful names to be able to use these errors from external packages` (or something like this).